tests/wsl/firstrun.pm

```diff
@@ -137,15 +144,23 @@ sub run {
         assert_screen 'wsl-image-ready-prompt', 120;
     }
 
-    unless (get_var('BETA')) {
-        become_root;
+    # Nothing to do in WSL2 pts w/o serialdev support
+    # https://github.com/microsoft/WSL/issues/4322
+    if (get_var('WSL2')) {
+        type_string "exit\n";
+        return;
     }
 
+    become_root unless (is_sut_dev() && is_sut_reg());
+
     assert_script_run 'cd ~';
     if (script_run "zypper ps") {
```

Comment:
is this correct? Looks like it should be treated like `unless`...

Reply:
It is this situation, when `lsof` package is missing
https://openqa.opensuse.org/tests/1390134#step/firstrun/19